metadata-ingestion/setup.py

```diff
@@ -140,6 +140,7 @@
     #    This was derived from work done by @jskrzypek in
     #    https://github.com/datahub-project/datahub/issues/8115#issuecomment-2264219783
     "acryl-great-expectations==0.15.50.1",
+    "jupyter_server>=2.14.1",
```

Comment:
this bump is because of a CVE, right?
it would be nice to add a comment mentioning that